trustedapps.json

```
@@ -8,6 +8,18 @@
       "identifier": "com.sidestoreapps.community",
       "sourceURL": "https://community-apps.sidestore.io/sidecommunity.json"
     },
+    {
+      "identifier": "com.aoshuang.manicemu",
+      "sourceURL": "https://apps.manicemu.site/altstore"
+    },
+    {
+      "identifier": "com.livecontainer.source",
+      "sourceURL": "https://raw.githubusercontent.com/LiveContainer/LiveContainer/refs/heads/main/apps.json"
+    },
+    {
+      "identifier": "com.stossy11.MeloNX",
+      "sourceURL": "https://git.ryujinx.app/melonx/emu/-/raw/XC-ios-ht/source.json?ref_type=heads"
```

```suggestion
      "sourceURL": "https://git.ryujinx.app/melonx/emu/-/raw/XC-ios-ht/source.json"
```
Maybe would be best to do this instead? (they are probably both the same tho)